src/helpers/getUniquePost.js

```diff
@@ -0,0 +1,11 @@
+import { db } from "../firebaseConfig"
+
+async function getUniquePost(id) {
+  return await db
+    .collection("posts")
+    .doc(id)
+    .get()
+    .then((post) => post.data())
+}
+
```

Comment:
Don't nee a separate file for each function.
It's not a good idea for helpers to export default.
Have one file `src/helpers/database.js` and put all these functions and export there.

Reply:
Maybe if you add comment related stuff to we can create a separate file, so each domain entity has a helper file.
`src/helpers/database/post.js` and `src/helpers/database/comment.js`.